.github/workflows/ci-test-backend.yml

```diff
@@ -8,13 +8,11 @@ on:
       - '.github/workflows/ci-test-backend.yml'
       - 'backend/**'
       - '!backend/scripts/**'
-      - '.golangci.yml'
```

Comment:
Why would you want to prevent tests from running on `.golangci.yml` change?

Reply:
To clarify, this line is intentionally added there so when linter configuration is changed, it's tested by CI as well.

Reply:
This file was removed from repo [here](https://github.com/umputun/remark42/commit/10c1aa939f5ad5c71e17e62978ca835e2dc16664), what is the reason to leave this line?

Reply:
backend/** [matches](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet) hidden files as well, all good then. Thank you!